Guidelines.md

```diff
@@ -211,11 +216,9 @@ https://api.contoso.com/v1.0/items?url=https://resources.contoso.com/shoes/fancy
 ```
 
 ### 7.2 URL length
-The HTTP 1.1 message format, defined in RFC 7230, in section [3.1.1][rfc-7230-3-1-1], defines no length limit on the Request Line, which includes the target URL. From the RFC:
+The HTTP 1.1 protocol, defined in RFC 2616, in section [3.2.1][rfc-2616-3-2-1], defines no URL length limit. From the RFC:
```

Comment:
:exclamation: I believe this was correct before (it reverts a change made in #41)